source/geometry_model/two_merged_chunks.cc

```diff
@@ -0,0 +1,1090 @@
+      // is used
+      return aspect::Utilities::Coordinates::CoordinateSystem::spherical;
+    }
+
+
+
+    template <int dim>
+    Point<dim>
+    TwoMergedChunks<dim>::natural_to_cartesian_coordinates(const std::array<double,dim> &position_tensor) const
+    {
+      // Ignore the topography to avoid a loop when calling the
+      // AsciiDataBoundary for topography which uses this function....
+      Point<dim> position_point;
+      for (unsigned int i = 0; i < dim; i++)
+        position_point[i] = position_tensor[i];
+      const Point<dim> transformed_point = manifold.push_forward_sphere(position_point);
+
+      return transformed_point;
+    }
+
+
+
+    template <int dim>
+    void
+    TwoMergedChunks<dim>::
+    declare_parameters (ParameterHandler &prm)
+    {
+      prm.enter_subsection("Geometry model");
+      {
+        prm.enter_subsection("Chunk with lithosphere boundary indicators");
+        {
+          prm.declare_entry ("Chunk inner radius", "0.",
+                             Patterns::Double (0.),
+                             "Radius at the bottom surface of the chunk. Units: \\si{\\meter}.");
+          prm.declare_entry ("Chunk outer radius", "1.",
+                             Patterns::Double (0.),
+                             "Radius at the top surface of the chunk. Units: \\si{\\meter}.");
+          prm.declare_entry ("Chunk grid merge radius", "1",
+                             Patterns::Double (0),
+                             "Radius at the top surface of the lower chunk, "
+                             "where the two grids merge. Units: \\si{\\meter}.");
+
+          prm.declare_entry ("Chunk minimum longitude", "0.",
+                             Patterns::Double (-180., 360.), // enables crossing of either hemisphere
+                             "Minimum longitude of the chunk. Units: degrees.");
+          prm.declare_entry ("Chunk maximum longitude", "1.",
+                             Patterns::Double (-180., 360.), // enables crossing of either hemisphere
+                             "Maximum longitude of the chunk. Units: degrees.");
+
+          prm.declare_entry ("Chunk minimum latitude", "0.",
+                             Patterns::Double (-90., 90.),
+                             "Minimum latitude of the chunk. This value is ignored "
+                             "if the simulation is in 2d. Units: degrees.");
+          prm.declare_entry ("Chunk maximum latitude", "1.",
+                             Patterns::Double (-90., 90.),
+                             "Maximum latitude of the chunk. This value is ignored "
+                             "if the simulation is in 2d. Units: degrees.");
+
+          prm.declare_entry ("Outer radius repetitions", "1",
+                             Patterns::Integer (1),
+                             "Number of cells in radial direction for the upper chunk.");
+          prm.declare_entry ("Grid merge radius repetitions", "1",
```

Comment:
Would `Inner chunk radius repetitions` and `Outer chunk radius repetitions` be better?